.claude/skills/pr/scripts/info.py

```
@@ -0,0 +1,91 @@
+        "pr", "checks", str(pr), "--json", "name,bucket,link", status_codes=(8,)
+    )
+    owner, name = repo()
+    _, threads = fetch_threads(owner, name, pr)
+    reviews = fetch_reviews(owner, name, pr)
+
+    state = view["state"] + (" (draft)" if view["isDraft"] else "")
+    labels = ", ".join(label["name"] for label in view["labels"])
+    print(f"#{view['number']} {view['title']}")
+    print(
+        f"{state} · {view['headRefName']} → {view['baseRefName']} · {view['headRefOid'][:8]}"
+        f" · by {login(view)} · +{view['additions']} −{view['deletions']} in {view['changedFiles']} files"
```

Ruff (0.16.3)

[warning] String contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)